es6-template-literals/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+<body>
+  <script>
+    const number = 9200;
+    const street = ' Irvine Center Drive';
+    const address = `${number} ${street}`;
+
+    console.log('LearningFuze:', address);
```

Comment:
This conversion isn't one-to-one, so it's producing a slightly different output:

---

![image](https://user-images.githubusercontent.com/16143771/127094836-005a5571-2165-49a7-a8d2-1988cdc79f18.png)

---

![image](https://user-images.githubusercontent.com/16143771/127094873-9767968a-38f2-487a-96e9-a543e891679a.png)

Reply:
![Screen Shot 2021-07-26 at 10 26 56 PM](https://user-images.githubusercontent.com/24905313/127095451-164ac561-59bf-4bf5-a076-db82464c8a06.png)